modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinBulkLoadSelfTest.java

```diff
@@ -89,6 +94,31 @@
     /** JDBC statement. */
     private Statement stmt;
 
+    /** Parametrized run param : cacheMode */
+    @Parameterized.Parameter(0)
+    public CacheMode cacheMode;
+
+    /** Parametrized run param : atomicity */
+    @Parameterized.Parameter(1)
+    public CacheAtomicityMode atomicityMode;
+
+    /** Parametrized run param : near mode */
+    @Parameterized.Parameter(2)
+    public Boolean isNear;
+
+    /** Test run configurations: Cache mode, atomicity type, is near */
+    @Parameterized.Parameters
+    public static Collection<Object[]> runConfig() {
+        return Arrays.asList(new Object[][] {
+            { PARTITIONED, CacheAtomicityMode.ATOMIC, true },
```

Comment:
We should use static import in both cases

Reply:
Fixed